internal/node/eventbroadcaster.go

```diff
@@ -0,0 +1,35 @@
+package node
+
+import (
+	"context"
+	"time"
+
+	"go.sia.tech/renterd/webhooks"
+	"go.uber.org/zap"
+)
+
+type (
+	EventBroadcaster interface {
+		BroadcastEvent(e webhooks.EventWebhook)
+	}
+
+	eventBroadcaster struct {
+		broadcaster webhooks.Broadcaster
+		logger      *zap.SugaredLogger
+	}
+)
+
+func NewEventBroadcaster(b webhooks.Broadcaster, l *zap.Logger) EventBroadcaster {
+	return &eventBroadcaster{
+		broadcaster: b,
+		logger:      l.Sugar().Named("events"),
+	}
+}
+
+func (eb *eventBroadcaster) BroadcastEvent(e webhooks.EventWebhook) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	if err := eb.broadcaster.BroadcastAction(ctx, e.Event()); err != nil {
```

Comment:
Yeah I considered that myself already. It's very thin indeed but I wanted the logger namespace and didn't want to add the concept of events to the webhooks manager per se. I'll change it though.

Reply:
I'll undo the move and bring back the internal `bus` package. I was expecting a comment on it but sometimes I just can't get over the `ibus` import. I'd like to keep the `EventBroadcaster`, thin as it may be, removing it just made me realise why I added it in the first place. It dirties up the mocks and the `Broadcaster` interface.

Reply:
You don't need to worry about that interface if you make it a helper.

I pushed a commit to show you what I mean. This also gets rid of the interface for events since those were a bit of an awkward workaround for constructors imo.